packages/plugins/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -0,0 +1,131 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
```

Comment:
It appears this is based on an old template. `./add_gem_package.sh redhat_access foreman_plugin foreman-plugins-nightly-rhel7` crashes on the empty license tag, but it does generate a cleaner base. For example, it builds and installs apipie docs but also runs the DB migration and seeds.